site-src/geps/overview.md

```diff
@@ -0,0 +1,123 @@
+
+Gateway Enhancement Proposals (GEPs) serve a similar purpose to the [KEP][kep]
+process for the main Kubernetes project:
+
+1. Ensure that changes to the API follow a known process and discussion
+  in the OSS community.
+1. Make changes and proposals discoverable (current and future).
+1. Document design ideas, tradeoffs, decisions that were made for
+  historical reference.
+
+## Process
+
+### 1. Circulate the Idea
+Before creating a GEP, share your high level idea with the community. This can
+be in one of many forms:
+
+- A [new GitHub Discussion](https://github.com/kubernetes-sigs/gateway-api/discussions/new)
+- On our [Slack Channel](https://kubernetes.slack.com/archives/CR0H13KGA)
+- On one of our [community meetings](contributing/community/#meetings)
+
+### 2. Agree on the Goals
+Although it can be tempting to start writing out all the details of your
+proposal, it's important to first ensure we all agree on the goals. The first
+version of your GEP should aim for a "Provisional" status and leave out any
+implementation details, focusing primarily on "Goals" and "Non-Goals".
+
+### 3. Document Implementation Details
+Now that everyone agrees on the goals, it is time to start writing out your
+proposed implementation details. These implementation details should be very
+thorough, including the proposed API spec, and covering any relevant edge cases.
+Note that it may be helpful to use a shared doc for part of this phase to enable
+faster iteration on potential designs.
+
+It is likely that throughout this process, you will discuss a variety of
+alternatives. Be sure to document all of these in the GEP, and why we decided
+against them. At this stage, the GEP should be targeting the "Implementable"
+stage.
+
+### 4. Implement the GEP as "Experimental"
+With the GEP marked as "Implementable", it is time to actually make those
+proposed changes in our API. In some cases, these changes will be documentation
+only, but in most cases, some API changes will also be required. It is important
+that every new feature of the API is marked as "Experimental" when it is
+introduced. Within the API, we use `<gateway:experimental>` tags to denote
+experimental fields.
+
+Before these changes are released, they will also need to be documented.
+GEPs that have not been both implemented and documented before a release
+cut off will be excluded from the release.
+
+### 5. Graduate the GEP to "Standard"
+Once this feature has met the [graduation criteria](/concepts/versioning/#graduation-criteria), it is
+time to graduate it to the "Standard" channel of the API. Depending on the feature, this may include
+any of the following:
+
+1. Graduating the resource to beta
+2. Graduating fields to "standard" by removing `<gateway:experimental>` tags
+3. Graduating a concept to "standard" by updating documentation
+
+## Status
+
+Each GEP has a status field that defines it's current state. Most GEPS will
+proceed through the following states:
```

Comment:
It would be helpful to add a line describing how state changes should happen - as a PR to update a merged GEP? As a subsequent, separate `Implementable` GEP after an initial `Provisional` GEP has been merged?

Reply:
I agree that documenting how state changes happen is important here; I think we should make it clear that a PR to update the state, probably after the GEP has been discussed at a community meeting, is the goal.

Reply:
Updated, thanks!